relay-server/src/services/processor/span/processing.rs

```diff
@@ -203,13 +218,19 @@ pub fn process(
         ItemAction::Keep
     });
 
-    if let Some(outcome) = sampling_result.into_dropped_outcome() {
-        state.managed_envelope.track_outcome(
-            outcome,
-            DataCategory::SpanIndexed,
-            dynamic_sampling_dropped_spans,
+    if sampling_decision.is_drop() {
+        relay_log::trace!(
+            span_count,
+            ?sampling_result,
+            "Dropped spans because of sampling rule",
```

Comment:
I assume this log was moved here so there is only one log entry instead of one per span?